bot/cogs/example_cog.py

```diff
@@ -27,10 +31,17 @@ def __init__(self, bot):
         self.bot = bot
         self._last_member = None
 
-    #To createa a command you decorate an async method with commands.command
```

Comment:
The typo with create has been bugging me since I started writing the translate cog lol. It needs to be fixed